packages/workbox-core/utils/assert.mjs

```diff
@@ -1,11 +1,27 @@
+import WorkboxError from '../models/WorkboxError.mjs';
+
 /*
  * This method returns true if the current context is a service worker.
  */
-const isSWEnv = () => {
-  return (typeof ServiceWorkerGlobalScope !== 'undefined' &&
-    self instanceof ServiceWorkerGlobalScope);
+const isSWEnv = (moduleName) => {
```

Comment:
Minor nit I noticed today, Google JS style says this should be named `isSwEnv`. I think we do this in other places as well, and in general I think we should be consistent with our style guide.